nginx-controller/nginx/configurator.go

```diff
@@ -280,7 +280,19 @@ func (cnf *Configurator) createConfig(ingEx *IngressEx) Config {
 
 	//Override from annotation
 	if lbMethod, exists := ingEx.Ingress.Annotations["nginx.org/lb-method"]; exists {
-		ingCfg.LBMethod = lbMethod
+		if cnf.isPlus() {
+			if parsedMethod, err := ParseLBMethodForPlus(lbMethod); err != nil {
+				glog.Errorf("Ingress %s/%s: Invalid value for the nginx.org/lb-method: got %q", ingEx.Ingress.GetNamespace(), ingEx.Ingress.GetName(), lbMethod)
```

Comment:
could you print the returned err in the logs?
```go
glog.Errorf("Ingress %s/%s: Invalid value for the nginx.org/lb-method: got %q", ingEx.Ingress.GetNamespace(), ingEx.Ingress.GetName(), lbMethod)
```
to
```golang
glog.Errorf("Ingress %s/%s: Invalid value for the nginx.org/lb-method, got %q: %v", ingEx.Ingress.GetNamespace(), ingEx.Ingress.GetName(), lbMethod, err)
```
Please do that here in Configurator and also in Controller